pkg/solana/chain.go

```diff
@@ -280,14 +281,19 @@ func newChain(id string, cfg *config.TOMLConfig, ks core.Keystore, lggr logger.L
 	// txm will default to sending transactions using a single RPC client if sendTx is nil
 	var sendTx func(ctx context.Context, tx *solanago.Transaction) (solanago.Signature, error)
 
-	if cfg.MultiNode.Enabled() {
-		chainFamily := "solana"
+	chainFamily := "solana"
```

Comment:
Not an issue for this PR but maybe we should have a centralized place where we maintain these chain family names.

Reply:
yeah you can use chainsel.FamilySolana